CMP1903_A1_2324/Game.cs

```diff
@@ -17,6 +17,24 @@ internal class Game
          */
 
         //Methods
-      
+        public int Roll()
+        {
+            //Calls for the Die Class for each Dice object
+            Die dice1 = new Die();
+            Die dice2 = new Die();
+            Die dice3 = new Die();
+
+            //Calls for the value stored and prints them for each dice object
```

Comment:
Print out dice rolls numbers for each one to make it clearer to read and more neat.